### PR TITLE
Issue 30. update RTD for pypi

### PR DESCRIPTION
**Pull request by yhan818**
**Description**

Update RTD documentation for upgrading necessary files and publishing to pypi.org

See #

**ToDo List**

 - [x] Update /docs/source/installation.rst

**Test plan**

- [x] Test https://redata-commons--31.org.readthedocs.build/en/31/installation.html

*Resources*

https://docs.python-guide.org/writing/structure/

*Screenshots or additional context*

### CODE REVIEW

**Review by zoidy — COMMENTED** [review]
@yhan818 Let's try to link PRs to issues explicitly (in the PR template there is a spot to insert a link to the corresponding issue). This makes it easier to quickly navigate to the corresponding issue.

File: docs/requirements.txt [hunk 1]
```diff
@@ -3,4 +3,5 @@ sphinx-rtd-theme==0.5.1
 sphinx-autodoc-typehints==1.11.1
 
 # for logger module
-pandas==1.4.3
+pandas==1.2.3
```
[the remark this answers is not in the document]
Comment (yhan818) [reply docs/requirements.txt:6]: Yes. Before we solved the issue. This is where Python 3.7 (see https://github.com/UAL-RE/redata-commons/issues/32 ) cannot find Pandas 1.4.3.

[the remark this answers is not in the document]
Comment (yhan818) [reply docs/requirements.txt:6]: I will create a new issue and PR for sphinx package updates.

**Review by zoidy — APPROVED** [review]
Fixes #30

**Pull request merged**